### PR TITLE
docs: add ant.design link

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

Closes https://github.com/eggjs/egg/issues/492

### CODE REVIEW

**Comment by popomore** [comment]
https://github.com/eggjs/doctools/pull/19

File: package.json [hunk 1]
```diff
@@ -64,6 +64,7 @@
     "koa-static": "^3.0.0",
     "mz-modules": "^1.0.0",
     "pedding": "^1.1.0",
+    "runscript": "^1.2.1",
```
Comment (dead-horse) [inline package.json]: 多了一个 runscript ?

Comment (popomore) [inline package.json]: 是不是之前漏了，跑 autod 出来的

Comment (fengmk2) [inline package.json]: https://github.com/eggjs/egg/blob/057bc47e4c5e3ec8faae0de3807f656fa4ef41d4/test/doc.test.js#L6

果然是

**Pull request merged**